src/clj/fluree/db/json_ld/ledger.cljc

```diff
@@ -86,9 +86,7 @@
           "http://www.w3.org/ns/shacl#qualifiedMinCount"        const/sh_qualifiedMinCount
           "http://www.w3.org/ns/shacl#qualifiedMaxCount"        const/sh_qualifiedMaxCount
           "http://www.w3.org/ns/shacl#qualifiedValueShapesDisjoint" const/sh_qualifiedValueShapesDisjoint
-          ;; fluree
-          const/iri-role                                        const/$f:role
-          const/iri-target-class                                const/$fluree:targetClass}))
```

Comment:
Can you remind me what we used `targetClass` and `role` for? And is removing these from our predefined properties backwards compatible?

Reply:
These were related to the old policy functionality which is replaced by policy queries.